src/sentry/static/sentry/app/views/settings/organizationTeams/teamMembers.tsx

```diff
@@ -256,35 +265,39 @@ class TeamMembers extends React.Component {
         )}
       </DropdownAutoComplete>
     );
-  };
+  }
 
-  removeButton = member => (
-    <Button
-      size="small"
-      icon={<IconSubtract size="xs" isCircled />}
-      onClick={this.removeMember.bind(this, member)}
-      label={t('Remove')}
-    >
-      {t('Remove')}
-    </Button>
-  );
+  removeButton(member: Member) {
+    return (
+      <Button
+        size="small"
+        icon={<IconSubtract size="xs" isCircled />}
+        onClick={() => this.removeMember(member)}
+        label={t('Remove')}
```

Comment:
When we pass a child of type string the button  component will automatically have a label ... so you don't need to pass it here 😉

```suggestion
```